### ISSUE TITLE
Why does the `friends` relationship show in the example query response?

### ISSUE THREAD

**Issue by ntjess** [aplbrain/grand-cypher#50]
From the example: https://github.com/aplbrain/grand-cypher/blob/5340cf793c9ad20c065031c76ffcae1caaf0f721/examples.md?plain=1#L28

The query is asking to only filter for `r:paid`, so why is `r:friends` showing in the list of returned edges? Or am I misunderstanding the expected return value?

**Comment by j6k4m8** [comment]
Indeed you're right that this is a bug — I think it is related to the fix in https://github.com/aplbrain/grand-cypher/pull/48 by @jackboyla (tagging for awareness!) which — as I'm looking at it now — I don't think has actually been deployed! My fault!!

With the code from the current `main` branch, this now returns:

```py
{
    "n.name": ["Alice", "Bob"],
    "m.name": ["Bob", "Alice"],
    "r.amount": [
        {(0, "paid"): 12, (1, "paid"): 40},
        {(0, "paid"): 6, (1, "paid"): None},
    ],
}

as expected!

---

I will roll v0.10.0 in a moment, which will include this fix!
```

**Comment by j6k4m8** [comment]
`v0.10.0` ([[PyPI](https://pypi.org/project/grand-cypher/0.10.0/)] [[GitHub Release](https://github.com/aplbrain/grand-cypher/releases/tag/v0.10.0)]) is now live and can be installed — this should resolve this issue! Let me know if the issue remains; I have also updated [the documentation of examples](https://github.com/aplbrain/grand-cypher/blob/master/docs/examples.md#multigraph) with the latest (actual) results!

**Comment by jackboyla** [comment]
I should have updated the examples - my bad! Thanks for flagging @ntjess and thanks for fixing @j6k4m8 🤗

**Comment by ntjess** [comment]
Thanks for the fast fix! Verified things are working on 0.10.0.

**Comment by j6k4m8** [comment]
No worries @jackboyla, my fault for not being on top of that!! I totally failed to roll your fixes!! 🙄 

Glad this worked @ntjess, thank you for the report and the testing!!
